main.go

```diff
@@ -50,15 +52,12 @@ var (
 )
 
 func init() {
-	envLogLevel, ok := os.LookupEnv("HNY_LOG_LEVEL")
-	if !ok {
-		envLogLevel = "info"
+	debug := envOrElseBool("HNY_DEBUG", false)
```

Comment:
I see what you mean about log levels. A boolean "debug or no?" should be fine.

I feel bad starting out with `HNY_`. If we're changing the environment variable anyway, what do you think about `HONEYCOMB_` to get towards the new "standard-ish" of environment variable naming? Anything we ought to do about accepting the previous incarnation of the env var or let it go and rest in peace?

Also, could this line move up into `var { }` so that it is declared and looked up only once? (… just in case we rename it again 😬 )

Reply:
AFAIK we didn't publicize the short-lived HNY_LOG_LEGEL at all, so I'm inclined to just let it go -- there's an easy drop-in replacement, and people wouldn't be turning it on all the time

Reply:
Bye HNY_LOG_LEVEL!

![We threw it on the groud!](https://user-images.githubusercontent.com/517302/130686710-4e385a51-5fdc-4fb0-b80b-6568c2e19550.gif)